website/docs/r/sql_database.html.markdown

```diff
@@ -37,6 +37,10 @@ The following arguments are supported:
 
 * `instance` - (Required) The name of containing instance.
 
+* `charset` - (Optional) The MySQL charset value.
```

Comment:
Nit: These entries should live under the horizontal divider with other optional fields.